sdks/python/apache_beam/io/gcp/bigquery_schema_tools.py

```diff
@@ -76,9 +76,9 @@ def generate_user_type_from_bq_schema(the_table_schema):
       raise ValueError(
           f"Encountered "
           f"an unsupported type: {the_schema['fields'][i]['type']!r}")
-    # TODO svetaksundhar@: Map remaining BQ types
-    dict_of_tuples.append((the_schema['fields'][i]['name'], typ))
-  sample_schema = beam.typehints.schemas.named_fields_to_schema(dict_of_tuples)
+    # TODO @svetaksundhar: Map remaining BQ types
```

Comment:
I think we can delete this TODO as it's already specified [here](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/io/gcp/bigquery_schema_tools.py#LL51C5-L52C39)

Reply:
Deleted this line as suggested